host/rootfs/usr/bin/setup.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+export WAYLAND_DISPLAY=wayland-1
+export XDG_RUNTIME_DIR=/run/user/0
+export WESTON_CONFIG_FILE=/etc/xdg/weston/weston.ini
+
+echo "12345678901264758695847364857456" > /etc/machine-id
+mkdir -p /home/apprunner
+mkdir /tmp && chmod 0777 /tmp
+adduser apprunner -D -h /home/apprunner
+chown apprunner:apprunner /home/apprunner
+chown -R apprunner:apprunner /run/user/0
+chown apprunner /dev/pts/0
+chmod -R 0755 /run/user/0
+chmod 666 /dev/null
```

Comment:
`666` is just bad practice even in demo. Would 660 with `root:apprunner`-ownership of these `/dev`-files work?

Reply:
It is not a bad practice, it is default access mode for Linux systems. Changing ownership of that files is a bad practice.

Reply:
for everything but `/dev/dri/card0` (`660`) you're right - haven't looked into the default access mode lately until now.
NixOS says:
```
❯ ls -la /dev/urandom
crw-rw-rw- 1,9 root 25 Oct 09:01 /dev/urandom

❯ ls -la /dev/tty
crw-rw-rw- 5,0 root 25 Oct 09:01 /dev/tty
❯ ls -la /dev/dri/card0
crw-rw----@ 226,0 root 25 Oct 15:50 /dev/dri/card0
❯ ls -la /dev/dri/renderD128
crw-rw-rw- 226,128 root 25 Oct 09:01 /dev/dri/renderD128
❯ ls -la /dev/null
crw-rw-rw- 1,3 root 25 Oct 09:01 /dev/null
```

When it's the default access mode - why do we need to `chmod` them? It just does not look right without comment on why.